app/assets/javascripts/student_profile/ProfilePdfDialog.js

```diff
@@ -17,8 +17,8 @@ export default class ProfilePdfDialog extends React.Component {
     const nowMoment = context.nowFn();
     this.state = {
       includeRestrictedNotes: false,
-      filterFromDate: firstDayOfSchool(toSchoolYear(nowMoment)-1),
-      filterToDate: nowMoment
+      filterFromDateText: firstDayOfSchool(toSchoolYear(nowMoment)-1).format('MM/DD/YYYY'),
+      filterToDateText: nowMoment.format('MM/DD/YYYY')
```

Comment:
Looking at this now, the end date should probably have been "now+1 day" also.  Otherwise it would cut off notes etc. from the current day.   If that seems right to you, want to add it in while we're in here?